src/main/java/svc/controllers/CitationController.java

```diff
@@ -54,25 +44,26 @@ Citation GetCitation(@PathVariable("id") Long id) {
 		if (citationNumber != null) {
 			criteria.citationNumber = citationNumber;
 		}
-		
+
 		if (dob != null) {
 			criteria.dateOfBirth = dob;
 		}
-		
+
 		if (licenseNumber != null && licenseState != null) {
 			criteria.driversLicenseNumber = licenseNumber;
 			criteria.driversLicenseState = licenseState;
 		}
-		
+
 		if (lastName != null && municipalityIds != null && municipalityIds.size() != 0) {
 			criteria.lastName = lastName;
 			criteria.municipalities = new ArrayList<Long>();
 			for(String municipalityId : municipalityIds){
 				criteria.municipalities.add(hashUtil.decode(Municipality.class,municipalityId));
 			}
 		}
-		
-		if (firstName != null && lastName != null &&  licenseNumber != null) { //for the text/phone system
+
+		// for the text system
```

Comment:
Sure. I'm not sure how that got changed, honestly. The comment was there but somehow I moved and changed it slightly a few weeks back.